lib/features/prediction/data/repositories/repo_impl.dart

```diff
@@ -0,0 +1,29 @@
+
+import 'package:either_dart/either.dart';
+
+import '../../../../core/shared/shared.dart';
+import '../../prediction.dart';
+
+class PredictionRepositoryImpl implements PredictionRepository {
+   final NetworkInfo network;
+  final PredictionRemoteDataSource remote;
+
+  const PredictionRepositoryImpl({
+    required this.network,
+    required this.remote,
+  });
+
+  @override
+  Future<Either<Failure, PredictionEntity>> get prediction async {
```

Comment:
Fix this line based on the me method override

Reply:
Done